core/trino-main/src/main/java/io/trino/operator/scalar/DateTimeFunctions.java

```diff
@@ -85,6 +88,8 @@
     private static final int MILLISECONDS_IN_HOUR = 60 * MILLISECONDS_IN_MINUTE;
     private static final int MILLISECONDS_IN_DAY = 24 * MILLISECONDS_IN_HOUR;
     private static final int PIVOT_YEAR = 2020; // yy = 70 will correspond to 1970 but 69 to 2069
+    private static final Slice YmD = Slices.utf8Slice("%Y-%m-%d");
+    private static final long MILLIS_PER_DAY = 24 * 3600 * 1000;
```

Comment:
remove, reuse `io.trino.spi.type.Timestamps#MILLISECONDS_PER_DAY`

Reply:
done in d6f05bd8ec6e41ee32e70f8d6cfb128e447d74b0